core/src/main/java/rs/kunpero/vacation/service/VacationService.java

```diff
@@ -50,16 +51,19 @@
     private final MethodsClient methodsClient;
     private final String accessToken;
     private final Clock clock;
+    private final DateTimeFormatter dateTimeFormatter;
 
     @Autowired
     public VacationService(VacationInfoRepository vacationInfoRepository, MessageSourceHelper messageSourceHelper, Clock clock,
-                           UserStatusService userStatusService, Slack slack, @Value("${slack.access.token}") String accessToken) {
+                           UserStatusService userStatusService, Slack slack, @Value("${slack.access.token}") String accessToken,
+                           @Value("${date.pattern}") String datePattern) {
         this.vacationInfoRepository = vacationInfoRepository;
         this.userStatusService = userStatusService;
         this.clock = clock;
         this.messageSourceHelper = messageSourceHelper;
         this.methodsClient = slack.methods(accessToken);
```

Comment:
I've found that we can use bean _methodsClient_ from the context (it's initialized in the SlackConfig). Hence, may I ask you to remove @Autowired annotation from this class?

Reply:
I have removed @Autowired annotation. But are you sure about this?